### DISCUSSION TITLE
Pressed keys are sometimes not handled any more?

### DISCUSSION THREAD

**Discussion by rumpelsepp** [wezterm/wezterm#1894] [Q&A]
In the latest release pressed keys (such as arrow keys) are sometimes not handled any more. If I move the cursor around in neovim with the arrow keys and change quickly from e.g. pressed <kbd>Left</kbd> to pressed <kbd>Right</kbd> wezterm seems to stop processing key events. I enabled `debug_key_events` and I can confirm that `wezterm` doesn't recognize the pressed key events if I switch quickly from one key to another. The key release is recognized again.

I figure it is related to this https://wezfurlong.org/wezterm/config/key-tables.html. wezterm seems to somehow debounce my key presses. Can I disable the key-tables feature?

**Comment by wez** [comment]
Are you using Wayland, or some other OS/environment?

**Reply by rumpelsepp** [reply]
I am on Wayland; gnome on wayland to be precise.
